src/records.js

```diff
@@ -16,8 +16,9 @@ const getRecord = async (recordsContract, dataHash) => {
   };
 };
 
-const getAttestation = async (recordsContract, dataHash, attestator) =>
-  recordsContract.sigExists.call(dataHash, attestator);
+const getAttestation = async (
+  recordsContract, dataHash, attestator,
+) => recordsContract.sigExists.call(dataHash, attestator);
```

Comment:
Did you run prettier?

Reply:
I ran eslint, and it took me a while to pass the tests in circleci because of that